packages/smooth_app/lib/pages/product/nutrition_page_loaded.dart

```diff
@@ -652,59 +654,64 @@ class _NutritionPageLoadedState extends State<NutritionPageLoaded> {
     } else if (<dynamic>[
-          product.nutriments!.saturatedFatServing,
-          product.nutriments!.proteinsServing,
-          product.nutriments!.novaGroupServing,
-          product.nutriments!.energyServing,
-          product.nutriments!.carbohydratesServing,
-          product.nutriments!.caffeineServing,
-          product.nutriments!.calciumServing,
-          product.nutriments!.ironServing,
-          product.nutriments!.vitaminAServing,
-          product.nutriments!.vitaminB1Serving,
-          product.nutriments!.vitaminB2Serving,
-          product.nutriments!.vitaminB6Serving,
-          product.nutriments!.vitaminB9Serving,
-          product.nutriments!.vitaminB12Serving,
-          product.nutriments!.vitaminCServing,
-          product.nutriments!.vitaminDServing,
-          product.nutriments!.vitaminEServing,
-          product.nutriments!.vitaminKServing,
-          product.nutriments!.vitaminPPServing,
-          product.nutriments!.magnesiumServing,
-          product.nutriments!.phosphorusServing,
-          product.nutriments!.potassiumServing,
-          product.nutriments!.sodiumServing,
-          product.nutriments!.zincServing,
-          product.nutriments!.copperServing,
-          product.nutriments!.seleniumServing,
-          product.nutriments!.cholesterolServing,
-          product.nutriments!.butyricAcidServing,
-          product.nutriments!.caproicAcidServing,
-          product.nutriments!.caprylicAcidServing,
-          product.nutriments!.lauricAcidServing,
-          product.nutriments!.myristicAcidServing,
-          product.nutriments!.palmiticAcidServing,
-          product.nutriments!.stearicAcidServing,
-          product.nutriments!.oleicAcidServing,
-          product.nutriments!.linoleicAcidServing,
-          product.nutriments!.docosahexaenoicAcidServing,
-          product.nutriments!.eicosapentaenoicAcidServing,
-          product.nutriments!.erucicAcidServing,
-          product.nutriments!.monounsaturatedServing,
-          product.nutriments!.polyunsaturatedServing,
-          product.nutriments!.alcoholServing,
-          product.nutriments!.pantothenicAcidServing,
-          product.nutriments!.biotinServing,
-          product.nutriments!.chlorideServing,
-          product.nutriments!.chromiumServing,
-          product.nutriments!.fluorideServing,
-          product.nutriments!.iodineServing,
-          product.nutriments!.manganeseServing,
-          product.nutriments!.molybdenumServing,
-          product.nutriments!.omega3FatServing,
-          product.nutriments!.omega6FatServing,
-          product.nutriments!.transFatServing
+          product.nutriments!.getValue(Nutrient.saturatedFat, PerSize.serving),
+          product.nutriments!.getValue(Nutrient.proteins, PerSize.serving),
+          //product.nutriments!.getValue(Nutrient.novaGroup, PerSize.serving),
+          //product.nutriments!.getValue(Nutrient.energy, PerSize.serving),
```

Comment:
There is energyKCal & energyKJ

Should I use both, or which one

Reply:
I recommend energyKCal & energyKJ.
energy is in KJ and is computed from energyKJ, or from energyKCal if null.